src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php

```diff
@@ -115,6 +115,27 @@ protected function redirectToRoute(string $route, array $parameters = [], int $s
         return $this->redirect($this->generateUrl($route, $parameters), $status);
     }
 
+    /**
+     * Returns a RedirectResponse to last accessed url.
+     *
+     * @author Temuri Takalandze (ABGEO) <takalandzet@gmail.com>
```

Comment:
we don't put `@author` annotations at the method level (we even avoid adding it too much on class level). The git history is what tells us about authorship if needed (with much less maintenance overhead).